documentation/book/getting-started.adoc

```diff
@@ -19,14 +21,13 @@ endif::Downloading[]
 ifdef::InstallationAppendix[]
 When no existing Kubernetes cluster is available, `minikube` can be used to create a local cluster.
 More details can be found in xref:installing_kubernetes_cluster[Installing Kubernetes clusters].
-{ProductName} supports a few specific OpenShift-specific features. 
-If you need an OpenShift cluster to use these and none is available `minishift` can be used to create a local cluster.
+
+If you need to use OpenShift-specific features and have no OpenShift cluster `minishift` can be used to create a local cluster.
```

Comment:
Nobody needs to use the OpenShift specific features - especially if they do not have an OPenShift cluster. Would this be better as _If you want ..._?

Reply:
```suggestion
If you want to use OpenShift-specific features and have no OpenShift cluster `minishift` can be used to create a local cluster.
```